help/destinations/catalog/advertising/magnite-batch.md

```diff
@@ -161,19 +164,19 @@ On the "Configure a filename and export schedule for each audience" screen, you
 >
 > A Mapping ID or "NONE" is required for this destination.
 >
-> A Mapping ID should be provided when an audience has a pre-existing Segment ID previously known to Magnite Streaming. Otherwise, "NONE" should be used as the Mapping ID.
```

Comment:
Please illustrate this with some screenshots. One with an existing mapping and one with NONE.